src/main/interface/settings.c

```diff
@@ -684,6 +684,9 @@ const clivalue_t valueTable[] = {
     { "dshot_idle_value",           VAR_UINT16  | MASTER_VALUE, .config.minmax = { 0, 2000 }, PG_MOTOR_CONFIG, offsetof(motorConfig_t, digitalIdleOffsetValue) },
 #ifdef USE_DSHOT_DMAR
     { "dshot_burst",                VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_MOTOR_CONFIG, offsetof(motorConfig_t, dev.useBurstDshot) },
+#ifdef USE_DSHOT_TELEMETRY
+    { "dshot_telemetry",            VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_MOTOR_CONFIG, offsetof(motorConfig_t, dev.useDshotTelemetry) },
```

Comment:
Not quite happy with this name - UART based telemetry triggered by Dshot is also 'Dshot telemetry', so this will be confusing for users. How about `dshot_erpm_telemetry`?

Reply:
I understand the concern. But erpm telemetry is also offered on the serial line so just as confusing. Maybe dshot_line_telemetry? Also a bit lame. dshot_live_telemetry? Oh how about this: dshot_bidirectional or dshot_bidir? Those have the advantage that they can't be confused with other telemetry settings.

Reply:
how 'bout dshot_realtime_telemetry? It may not scientifically qualify for the term 'realtime', but at least it's a commonly well understood term...?

Reply:
done